lib/govuk_publishing_components/presenters/component_wrapper_helper.rb

```diff
@@ -146,6 +153,16 @@ def check_hidden_is_valid(hidden_attribute)
         end
       end
 
+      def check_tabindex_is_valid(tabindex_attribute)
+        return if tabindex_attribute.blank?
+
+        tabindex_attribute = tabindex_attribute.to_s
+
+        unless /^-?[0-9]+$/.match?(tabindex_attribute)
```

Comment:
I thought tabindex values should only be positive integers or `-1`, but this seems to be allowing other values, including numbers with underscores? Can you clarify?

Reply:
@andysellick MDN says you can pass any negative value:

> A negative value (the exact negative value doesn't actually matter, usually tabindex="-1") means that the element is not reachable via sequential keyboard navigation.

The underscored numbers are a result of our Rubocop linting, as that's how it wants to represent large numbers. I have checked how this renders in our frontend and it appears as a regular number.

However checking MDN, the maximum positive number it says is possible is for `tabindex` is `32767`. Do you want me to incorporate this check into the code?

Reply:
No, that's fine 👍